kratos/utilities/data_type_traits.h

```diff
@@ -194,6 +228,30 @@ template<class TDataType> class DataTypeTraits
         *pContiguousDataBegin = rValue;
     }
 
+    /**
+     * @brief Copies the given container element value to contiguous array.
+     *
+     * This method copies all the elements of @ref rContainer recursively to the
+     * provided @ref pContiguousDataBegin.
```

Comment:
Please run `doxygen` and check whether you get any warnings/errors for the files you changed.

In this particular case, you're trying to reference a function argument, which is **not** a symbol. `doxygen` will complain about this. Use `@p` or something similar to highlight the variable.

https://www.doxygen.nl/manual/commands.html

```suggestion
     * provided @p pContiguousDataBegin.
```

Reply:
fixed all